src/interfaces/IMorpho.sol

```diff
@@ -143,7 +143,7 @@ interface IMorpho {
     /// - The token can revert on `transfer` and `transferFrom` for a reason other than an approval or balance issue.
     /// - A very high amount of assets (~1e35) supplied or borrowed can make the computation of `toSharesUp` and
     /// `toSharesDown` overflow.
-    /// - The IRM can revert on `borrowRate`.
+    /// - The IRM can revert on `borrowRate` which means it could lead to funds being stuck.
```

Comment:
I believe it is the case for all of these liveness-breaking situations, so we should duplicate this comment everytime?

But I understand that "break Morpho's liveness properties" imply that funds may get stuck in some cases, so I would say this comment is simply highlighting that no liveness => funds may be stuck
Perhaps we want to change the wording of the sentence introducing the bullet points?

Reply:
That was an argument made in the related issue. We can acknowledge it also

Reply:
Agree with @Rubilmax, I would just add that funds **could** get stuck after "break Morpho's liveness properties"

Reply:
Ah, right: https://github.com/cantinasec/review-morpho-blue-1/issues/41#issuecomment-1780862265
I'm fine with both!